contracts/core/LyraVault.sol

```diff
@@ -59,13 +59,13 @@ contract LyraVault is Ownable, BaseVault {
   function trade(uint strikeId) external {
     require(vaultState.roundInProgress, "round closed");
     // perform trade through strategy
-    (uint positionId, uint premiumReceived, uint collateralAdded) = strategy.doTrade(strikeId, lyraRewardRecipient);
+    (uint positionId, uint premiumReceived, uint capitalUsed) = strategy.doTrade(strikeId, lyraRewardRecipient);
```

Comment:
return naming should stay consistent with the doTrade interface on L7 in IStrategy.sol